src/context/AppProvider/AppContext.tsx

```diff
@@ -221,22 +233,29 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
       })
     })()
   }, [
-    dispatch,
     portfolioLoadingStatus,
     portfolioAccounts,
-    portfolioAssetIds,
-    wallet,
-    requestedAccountIds,
     DynamicLpAssets,
+    dispatch,
+    requestedAccountIds,
+    portfolioAssetIds,
   ])
 
+  const uniV2LpIdsData = useAppSelector(selectUniV2LpIds)
+
   // once the portfolio is loaded, fetch market data for all portfolio assets
   // start refetch timer to keep market data up to date
   useEffect(() => {
     if (portfolioLoadingStatus === 'loading') return
 
+    // Exclude assets for which we are unable to get market data
+    // We would fire query thunks / XHRs that would slow down the app
+    // We insert price data for these as resolver-level, and are unable to get market data
```

Comment:
todo after packages can import lib - market service should dynamically blacklist these ids - rather than the consumer having knowledge of what it can/can't query

Reply:
stop gap - do this in the marketDataSlice. get these ids from the state and return no data without making a query.

Reply:
Let's do this in a follow-up? Same circular deps issues here, it's a bit sad

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/17035424/231468293-2bd78da8-dd89-4a2d-b126-accd0409da6a.png">

Reply:
agreed will be easier in follow up